pulser-core/pulser/register/_coordinates.py

```diff
@@ -41,12 +41,15 @@ def sorted_coords(self) -> pm.AbstractArray:
         # Copies to prevent direct access to self._sorted_coords
         return self._sorted_coords.copy()
 
+    @cached_property
+    def rounded_coords(self) -> pm.AbstractArray:
```

Comment:
I would prefer if this was a private property, unless you thought of a case where it makes sense to have it public